src/PointwiseFunctions/GeneralRelativity/Christoffel.hpp

```diff
@@ -107,6 +107,27 @@ struct TraceSpatialChristoffelFirstKindCompute
   using base = TraceSpatialChristoffelFirstKind<SpatialDim, Frame, DataType>;
 };
 
+/// Compute item for the trace of the spatial Christoffel symbols
+/// of the second kind
+/// \f$\Gamma^{i} = \Gamma^{i}_{jk}g^{jk}\f$ computed from the
```

Comment:
`g` or `gamma`? I think this is the spatial metric, right?

Reply:
yes, this `g` is the spatial metric. I've corrected it to `\gamma` here + another place in the same file.